portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+/* Class that represents how a comment object */
+
+public class Comment {
+  private final String username;
+  private String text;
+
+  public Comment(String username, String text) {
```

Comment:
Optional: When you get tired of writing boilerplate like this, you may want to look into AutoValue. https://github.com/google/auto/blob/master/value/userguide/index.md